src/camera/v4l2/SDL_camera_v4l2.c

```diff
@@ -1132,8 +1000,8 @@ static int MaybeAddDevice(const char *path)
     }
 
 
-    /* Add new item */
-    item = (SDL_capturelist_item *)SDL_calloc(1, sizeof(SDL_capturelist_item));
+    // Add new item
+    item = (SDL_cameralist_item *)SDL_calloc(1, sizeof(SDL_cameralist_item));
```

Comment:
I can't leave a comment on the line number, but `path` is `SDL_strdup`'ed twice, leading to a leak.